src/scripting_api/app.js

```diff
@@ -23,6 +23,7 @@ const VIEWER_TYPE = "PDF.js";
 const VIEWER_VARIATION = "Full";
 const VIEWER_VERSION = 21.00720099;
 const FORMS_VERSION = 21.00720099;
+const USERACTIVATION_CALBACKID = 0;
```

Comment:
Making this a constant was a good idea, but please fix the spelling :-)
```suggestion
const USERACTIVATION_CALLBACKID = 0;
```